### PR TITLE
Perso REST: ezp -> ibexa

### PR DESCRIPTION
**Pull request by reithor**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | 4.6, master
| Edition       | All

PR Fixes wrong namings (REST API)
See https://github.com/ibexa/personalization/blob/main/src/bundle/Resources/config/routing_rest.yaml for correct REST routes

Section about contenttypes endpoint needs to reevaluated - the endpoint does not exist in current release! 

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

**Comment by reithor** [comment]
Hi @ciastektk ,
Docs mention a `contenttypes` endpoint for the REST API (see
https://doc.ibexa.co/en/latest/personalization/enable_personalization/#check-whether-the-bundle-provides-rest-data) - which is not defined in https://github.com/ibexa/personalization/blob/main/src/bundle/Resources/config/routing_rest.yaml.

Was the endpoint removed at some point or is `/personalization/v1/content/list/{contentIds}`  a replacement ?

File: docs/personalization/enable_personalization.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+git check---
```
Comment (mnocon) [inline docs/personalization/enable_personalization.md:1]: Please remember to remove this 😄

**Review by mnocon — APPROVED** [review]
Thank you!

**Review by adamwojs — APPROVED** [review]

File: docs/personalization/enable_personalization.md (hunk 1 above)
Comment (reithor) [inline docs/personalization/enable_personalization.md:1]:
```suggestion
```

**Comment by mnocon** [comment]
@reithor About:

> Hi @ciastektk , Docs mention a `contenttypes` endpoint for the REST API (see https://doc.ibexa.co/en/latest/personalization/enable_personalization/#check-whether-the-bundle-provides-rest-data) - which is not defined in https://github.com/ibexa/personalization/blob/main/src/bundle/Resources/config/routing_rest.yaml.
> 
> Was the endpoint removed at some point or is `/personalization/v1/content/list/{contentIds}` a replacement ?

I see these endpoint was removed in https://github.com/ezsystems/ezrecommendation-client/pull/102 (cross-merged in https://github.com/ibexa/personalization/pull/117) and there is no replacement mentioned - it's described as useless code.

Could you please remove mentions of this endpoint and then we could merge this?

**Comment by reithor** [comment]
@mnocon @nkostiv :
- I haved added details about token authorization
- also I have changed the export command part (export command always needs siteaccess, customer-id and license-key
- we should consider adding information about where exported files ares stored locally (see also: https://github.com/ibexa/documentation-developer/pull/2499)

File: docs/personalization/enable_personalization.md [hunk 2]
```diff
@@ -161,33 +160,43 @@ Place the following code snippet in the `<head>` section of your header template
...
+
+You can use this token to check what is provided to the Personalization Engine:
 
-To check whether the `content` endpoint is working as expected, perform the following request:
 
 ```
-GET http://<yourdomain>/api/ezp/v2/ibexa_personalization/v1/content/{contentId}
-Accept application/vnd.ez.api.Content+json
-Authorization Basic xxxxxxxx
+curl --location '{PERSONALIZATION_HOST_URI}/api/ibexa/v2/personalization/v1/content/id/{contentId}?lang={comma_separated_languages}' \
```
Comment (adriendupuis) [inline docs/personalization/enable_personalization.md:176]: Does it need a domain dedicated to personalization? If yes, `PERSONALIZATION_HOST_URI` need a definition; if not, it need to me more generic. Why not `HOST_URI` or `YOUR_HOST_URI`?

Comment (nkostiv) [reply docs/personalization/enable_personalization.md:176]: @adriendupuis this is a variable mentioned during personalisation configuring process https://doc.ibexa.co/en/latest/personalization/enable_personalization/

**Review by nkostiv — APPROVED** [review]

**Review by adriendupuis — COMMENTED** [review]
Preview: https://ez-systems-developer-documentation--2346.com.readthedocs.build/en/2346/personalization/enable_personalization/#check-whether-the-bundle-provides-rest-data

Thank you @reithor  for the update and enhencement.

I'm not sure about the examples using `curl` format instead of HTTP format. I would have fix and update the HTTP request examples. At list, the new `curl` example is correct while previous HTTP syntax was wrong (wrong regarding HTTP syntax even before being outdated regarding perso REST).

File: docs/personalization/enable_personalization.md [hunk 3]
```diff
@@ -161,33 +160,43 @@ Place the following code snippet in the `<head>` section of your header template
...
+--header 'Accept: application/vnd.ibexa.api.Content+json' \
+--header 'Authorization: Bearer {BEARER_TOKEN}'
 ```
 
-Additionally, check whether the `contenttypes` endpoint is working with the following request:
+Additionally, check whether the `contentlist` endpoint is working with the following request:
 
 ```
```
[the remark this answers is not in the document]
Comment (reithor) [reply docs/personalization/enable_personalization.md:183]: @adriendupuis :
We have several places, that use cURL - in context of REST API and HTTP cache / Fastly chapters.
Also it makes sense to have something "usable" for own testing here.

**Review by adriendupuis — APPROVED** [review]
We have other HTTP request examples in the doc using `curl` instead of HTTP format. It is not too complicated for a developer to convert curl to another HTTP client. And it can be useful for a first test to study the answer. No change needed there.

**Pull request merged**